src/jupyterlabcontroller/constants.py

```diff
@@ -53,6 +57,12 @@
 gives up.
 """
 
+LAB_COMMAND = "/opt/lsst/software/jupyterlab/runlab.sh"
```

Comment:
Interesting.  I can no longer remember why I rely instead on setting things like NONINTERACTIVE inside the lab.